CHANGELOG.md

```diff
@@ -38,6 +35,15 @@
 
 ### Fixed
 
+## Forest v0.10.0 "Premature"
+
+### Breaking
+
+- [#3007](https://github.com/ChainSafe/forest/pull/3007): Optimize DB
+  parameters. This requires all existing databases to be re-initialized.
+
```

Comment:
Why is it in the diff? It should already be in the `main`, no?

Reply:
It was moved from `Unreleased` to `Forest v0.10.0 "Premature"`

Reply:
Ah, my bad, I expected the diff to show it differently.